information/gmt_data_server.txt

```diff
@@ -0,0 +1,59 @@
+# Dir			Name					xxy	reg	size    Remark
+#
+# Symbolic links for convenience and backwards compatibility with GMT <= 6.0.0; these points to the gridline-registered versions below
+#
+/		earth_relief_60m.grd	60m	g	128K	Earth Relief at 60x60 arc minutes from Gaussian Cartesian filtering (111 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_01d.grd	01d	g	128K	Earth Relief at 1x1 arc degrees from Gaussian Cartesian filtering (111 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_30m.grd	30m	g	435K	Earth Relief at 30x30 arc minutes from Gaussian Cartesian filtering (55 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_20m.grd	20m	g	918K	Earth Relief at 20x20 arc minutes from Gaussian Cartesian filtering (37 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_15m.grd	15m	g	1.6M	Earth Relief at 15x15 arc minutes from Gaussian Cartesian filtering (28 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_10m.grd	10m	g	3.4M	Earth Relief at 10x10 arc minutes from Gaussian Cartesian filtering (18 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_06m.grd	06m	g	8.8M	Earth Relief at 6x6 arc minutes from Gaussian Cartesian filtering (10 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_05m.grd	05m	g	 13M	Earth Relief at 5x5 arc minutes from Gaussian Cartesian filtering (9 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_04m.grd	04m	g	 19M	Earth Relief at 4x4 arc minutes from Gaussian Cartesian filtering (7.5 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_03m.grd	03m	g	 33M	Earth Relief at 3x3 arc minutes from Gaussian Cartesian filtering (5.6 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_02m.grd	02m	g	 71M	Earth Relief at 2x2 arc minutes from Gaussian Cartesian filtering (3.7 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_01m.grd	01m	g	258M	Earth Relief at 1x1 arc minutes from Gaussian Cartesian filtering (1.9 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_30s.grd	30s	g	935M	Earth Relief at 30x30 arc seconds from Gaussian Cartesian filtering (1.0 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/		earth_relief_15s.grd	15s	p	3.2G	Earth Relief at 15x15 arc seconds provided by SRTM15+V2.1 [Tozer et al., 2019]
+#
+# Special names for files based on tiles.  These are virtual files that never exist
+#
+/srtm3	earth_relief_03s.grd	03s	p	6.8G	Earth Relief at 3x3 arc seconds tiles provided by SRTMGL3 (land only) [NASA/USGS]
+/srtm1	earth_relief_01s.grd	01s	p	 41G	Earth Relief at 1x1 arc seconds tiles provided by SRTMGL1 (land only) [NASA/USGS]
+/srtm3	earth_relief_03s_p.grd	03s	p	6.8G	Earth Relief at 3x3 arc seconds tiles provided by SRTMGL3 (land only) [NASA/USGS]
+/srtm1	earth_relief_01s_p.grd	01s	p	 41G	Earth Relief at 1x1 arc seconds tiles provided by SRTMGL1 (land only) [NASA/USGS]
```

Comment:
Only pixel registration for srtm1 and srtm3? I'm OK with it, just to confirm if they're typos.

Reply:
Yes, those are the original files, in pixel registration only.  Cannot convert as per my earlier explanations. And because of tiling, not possible to sample a gridlines at the corners of a tile.